### PR TITLE
Update workouts extension

### PR DESCRIPTION
**Pull request by diannemcewan**
## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by samuelkraft** [comment]
@diannemcewan Nice improvement! What do you think about using a [List.EmptyView](https://developers.raycast.com/api-reference/user-interface/list#list.emptyview) for the success view? That would allow you to add a nice graphic and center the text. I know @pernielsentikaer did something similar for https://www.raycast.com/pernielsentikaer/hiit

**Comment by diannemcewan** [comment]
> @diannemcewan Nice improvement! What do you think about using a [List.EmptyView](https://developers.raycast.com/api-reference/user-interface/list#list.emptyview) for the success view? That would allow you to add a nice graphic and center the text. I know @pernielsentikaer did something similar for https://www.raycast.com/pernielsentikaer/hiit

I played a little with a list view instead of this, but the issue was that you see the search bar unnecessarily, unless you know a nice way to disable that?

**Comment by samuelkraft** [comment]
> > @diannemcewan Nice improvement! What do you think about using a [List.EmptyView](https://developers.raycast.com/api-reference/user-interface/list#list.emptyview) for the success view? That would allow you to add a nice graphic and center the text. I know @pernielsentikaer did something similar for https://www.raycast.com/pernielsentikaer/hiit
> 
> @samuelkraft - I agree I don't LOVE the markdown. I played a little with a list view instead of this, but the issue was that you see the search bar unnecessarily, unless you know a nice way to disable that?

I don't think that's possible unfortunately. If you think the markdown works better lets merge this 👍

File: extensions/workouts/src/createActivity.tsx [hunk 1]
```diff
@@ -41,16 +43,41 @@ function CreateActivity() {
       title: "Saving your activity",
     });
     try {
-      await createActivity({ ...values, distanceUnit: distanceUnitRef.current });
+      const activity = await createActivity({ ...values, distanceUnit: distanceUnitRef.current });
       toast.style = Toast.Style.Success;
       toast.title = "Activity saved";
+      setIsSubmitted(true);
+      console.log(activity.name);
```
Comment (samuelkraft) [inline extensions/workouts/src/createActivity.tsx:50]: remove console log

**Review by samuelkraft — APPROVED** [review]

**Pull request merged**